ansible/roles/ghostunnel/templates/etc/default/ghostunnel.j2

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2021 Pedro Luis Lopez <pedroluis.lopezsanchez@gmail.com>
+# Copyright (C) 2021 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
The copyright and license header should be put between `{#` and `#}` markers so that it's not included in the final configuration file rendered by Jinja.

Reply:
Done!